opendut-util/src/logging.rs

```diff
@@ -166,12 +232,31 @@ impl LoggingConfig {
         } else {
             None
         };
+        let opentelemetry_metrics_interval_ms:Option<Duration> = if opentelemetry_enabled {
```

Comment:
Can you create a configuration object (struct) for all those parameters required for the metrics?

```
#[derive(Clone, Debug, Deserialize)]
pub struct OpentelemetryConfig {
  pub  enable: bool,
  pub logging_stdout: bool,
  pub endpoint: Url,
  pub service_name: String,
  pub metrics_interval_ms: u64,
}
```
and parse it by implementing the following trait:
```
impl TryFrom<&Config> for OpentelemetryConfig {

}
```
This has been on several other occasions throughout the code base. For example here: 
https://github.com/eclipse-opendut/opendut/blob/d5c22ce0af03a0546efc800587b064e717a384c6/opendut-carl/opendut-carl-api/src/carl/auth/auth_config.rs#L41

This has the benefit of not having to pass all the parameters individually to all the methods but to have a single parameter, namely this config struct, that is being passed to the methods.